index.html

```diff
@@ -0,0 +1,153 @@
+                <span class="fa-brands fa-twitter"></span>
+                <h5> Blog </h5>
+                <h5> Sponsor </h5>
+                <h5>News </h5>
+
+            </nav>
+            <div class="desktop-nav">
+                <button type="button" class="ham-burger" id="desk">
+                    <span class="glyphicon glyphicon-menu-hamburger"></span>
+                </button>
+                <a href="#frontPage">
+                    <img class="desktop-logo" src="images/logo.jpeg" alt="Logo Image">
+                </a>
+                <div class="desktop-links">
+                    <a href="#frontPage" class=" link" id="red">Home</a>
+                    <a href="#mainPage" class="link" id="mid">About</a>
+                    <a href="" class=" link border" id="red">Contact Us</a>
+                </div>
+                <div class="mobile-links">
+
+                    <button type="button" class="cross">
+                        <span class="fa-solid fa-xmark"></span>
+                    </button>
+                    <a href="#frontPage" id="decoration" class=" mob-link mob-link1">Home</a>
+                    <a href="#mainPage" id="decoration" class=" mob-link mob-link2">About</a>
+                    <a href="" id="decoration" class=" mob-link mob-link3">Contact Us</a>
+                </div>
+            </div>
+        </nav>
+
+        <div class="desktop-padding">
+            <h4 class="heading2 front">Let's grow together!</h4>
+            <h1 class="heading3">Present Your Business In The Digital World With Us</h1>
+            <p class="font-description"> Get Started with us to help boost your online business. We help you with our
+                digital services and support you on your entry into the world of digital marketing.</p>
+            <h2 class="heading4">SET UP A FREE MEETING</h2>
+        </div>
+    </section>
+    <section class="main-page" id="mainPage">
+        <h3 class="service-text">Our Services</h3>
+        <hr id="main">
+        <div class="wrap">
+            <div class="service">
+                <div class="icon">
+                    <span class="fa-solid fa-users images"></span>
+                    <h3 class="icon-label">Social Media Marketing</h3>
+                </div>
+                <p class="icon-para">The advice offered by our <strong>Social Media Agency</strong> guarantees concrete
+                    and measurable results.</p>
+            </div>
+            <div class="service">
+                <div class="icon">
+                    <span class="fa-solid fa-chart-simple images"></span>
+                    <h3 class="icon-label">Pay-Per-Click (PPC)</h3>
+                </div>
+                <p class="icon-para"> <strong>PPC</strong> allows you to immediately present your offers on major search
+                    engines and social channels.</p>
+            </div>
+            <div class="service">
+                <div class="icon">
```

Comment:
- [ ] I would kindly suggest you limit the main project section (Our service) to be 5 cards as seen in the [Cindy Shin design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)